client/components/site-verticals-suggestion-search/index.jsx

```diff
@@ -26,35 +26,82 @@ import './style.scss';
 
 export class SiteVerticalsSuggestionSearch extends Component {
 	static propTypes = {
+		autoFocus: PropTypes.bool,
```

Comment:
We don't seen to be documenting `autoFocus` in the README - now is a good chance to do that.